File: sdkwork-app-sdk-csharp/Models/ExportVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class ExportVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? TaskId { get; set; }
        public string? FileUrl { get; set; }
        public string? FileName { get; set; }
        public int? FileSize { get; set; }
        public string? Status { get; set; }
        public string? Format { get; set; }
        public int? ExportCount { get; set; }
        public string? ExpireTime { get; set; }
    }
}

File: sdkwork-app-sdk-go/types/export_vo.go
package types

// 导出响应
type ExportVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	TaskId string `json:"taskId"`
	FileUrl string `json:"fileUrl"`
	FileName string `json:"fileName"`
	FileSize int `json:"fileSize"`
	Status string `json:"status"`
	Format string `json:"format"`
	ExportCount int `json:"exportCount"`
	ExpireTime string `json:"expireTime"`
}
